Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -729,6 +730,10 @@ function vtkOpenGLTexture(publicAPI, model) {
     model.depth = 1;
     model.numberOfDimensions = 2;
     model.openGLRenderWindow.activateTexture(publicAPI);
+    model.maxLevel = level;
```

Comment:
you can use directly `model.maxLevel = data.length / 6;` and get rid of the temporary variable.